### ISSUE TITLE
[BUG] Avorion avserver.cfg admins not used in config

### ISSUE THREAD

**Issue by kdulep** [GameServerManagers/LinuxGSM#3512]
## User Story
Command-line Parameters
====================================================================
 ./server.sh --galaxy-name avserver --ip 0.0.0.0 --datapath /home/avserver/serverfiles/galaxy

adminsteamid="XXXXXXX"
password= also never used

I want dedicated server use steam admin "XXXXXXX"

## Basic info

* **Distro:** [CentOS8]
* **Game:** [Avorion]
* **Command:** [Details]
* **LinuxGSM version:** [v21.2.4]

## Further Information

Seems avserver.cfg dont get used

in server log:
administrators:
Warning: Your server does not have an administrator assigned.

## To Reproduce

Steps to reproduce the behaviour:
1. Change avserver.cfg adminsteamid="1232321321321321"
2. Start server
3. See details dt
4. No admin option to exe

## Expected behaviour
admin passed to exe
"password" used in server

**Comment by kdulep** [comment]
https://termbin.com/6p8v

**Comment by Claiyc** [comment]
Avorion admins should be defined in serverfiles/admin.xml. See https://www.citadelservers.com/client/knowledgebase/183/Adding-Admins-to-your-Avorion-Server.html (Method 2). 

Not a bug.

**Comment by kdulep** [comment]
why thre option adminsteamid?

**Comment by Claiyc** [comment]
> why thre option adminsteamid? in code it should add admins to command line

You‘re right, that should work. Not sure why it doesn‘t.

**Comment by dgibbs64** [comment]
So I guess the question is. Should the adminsteamid in _default.cfg be removed since the xml option is probably the better method?

**Comment by Claiyc** [comment]
I guess yes. Imho it doesn‘t make much sense to be able to configure it in two different ways. That only adds an unnecessary layer of complexity.

**Comment by dgibbs64** [comment]
Makes sense. It won't be in for this release. So if you want to fix this one when you get back feel free 🙂

**Comment by Claiyc** [comment]
I put it on my todo list :)
